api/src/applications/applications.controller.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  Body,
+  Controller,
+  Get,
+  Post,
+  UsePipes,
+  ValidationPipe,
+  HttpCode,
+  HttpStatus,
+  Request,
+  Delete,
+  Param,
+  Put,
+} from '@nestjs/common';
+import { ApplicationsService } from './applications.service';
+import { ApplicationsDto } from './dto/applications.dto';
+
+@Controller('applications')
+export class ApplicationsController {
+  constructor(private applicationService: ApplicationsService) {}
+
+  @Get()
+  @HttpCode(HttpStatus.OK)
+  getApplications() {
```

Comment:
Add async keyword before the every method declaration would be necessary if the method returns a Promise.

Reply:
async keyword add all methods